modules/UI/videolayout/VideoContainer.js

```diff
@@ -55,8 +53,12 @@ function computeDesktopVideoSize( // eslint-disable-line max-params
         videoHeight,
         videoSpaceWidth,
         videoSpaceHeight) {
-    const aspectRatio = videoWidth / videoHeight;
+    if (videoWidth === 0 || videoHeight === 0 || videoSpaceWidth === 0 || videoSpaceHeight === 0) {
```

Comment:
does it make sense for videoSpaceWidth or videoSpaceHeight to be 0 if videoWidth and videoHeight are not?

Reply:
what do you mean? if any one of those is zero this is a problem generally... 

In the context of the function those are numeric params. So 0 makes sense for any of them I guess...

Reply:
I mean, how could videoSpaceWidth or videoSpaceHeight be zero? If videoWidth and videoHeight aren't, that is.

Reply:
Not sure. But does it matter?